### ISSUE TITLE
TEST: Add conv correctness test suite

### ISSUE THREAD

**Issue by antiagainst** [nod-ai/SHARK-Turbine#624]

**Comment by antiagainst** [comment]
https://github.com/openxla/iree/pull/16849 landed for this. @pashu123 is there more work planned?

**Comment by pashu123** [comment]
> [openxla/iree#16849](https://github.com/openxla/iree/pull/16849) landed for this. @pashu123 is there more work planned?

Yes, there's a minor work to add attributes to enable this for GPU.

**Comment by pashu123** [comment]
@antiagainst If this is a priority, I can add the feature.
